src/parser.ts

```diff
@@ -254,7 +256,7 @@ function createStatementParserByToken(token: Token, options: ParseOptions): Stat
         return createTruncateStatementParser(options);
       case 'DECLARE':
       case 'BEGIN':
-        if (options.dialect === 'oracle') {
+        if (['oracle', 'bigquery'].includes(options.dialect)) {
```

Comment:
For my own reference: https://cloud.google.com/bigquery/docs/reference/standard-sql/procedural-language